pkg/api/v1alpha1/edgeconnect/edgeconnect_types.go

```diff
@@ -214,9 +213,14 @@ func (e *EdgeConnect) HostPatterns() []string {
 	return hostPatterns
 }
 
-func (e *EdgeConnect) HostMappings() []edgeconnect.HostMapping {
-	hostMappings := make([]edgeconnect.HostMapping, 0)
-	hostMappings = append(hostMappings, edgeconnect.HostMapping{From: e.K8sAutomationHostPattern(), To: kubernetesDefaultDNS})
+type HostMapping struct {
+	From string `json:"from"`
+	To   string `json:"to"`
+}
```

Comment:
Shouldn't we keep this on the client package? Because it's only used through the API and it's not in the CR (for now).
But I don't have a strong opinion about it

Reply:
What I kind of disklike is that, we have an dependency from the API pkg downward.